ModelViewer/ModelViewer/ViewerGraphicsWindow.cpp

```diff
@@ -167,4 +167,32 @@ void ViewerGraphicsWindow::render()
     m_program->release();
 
     ++m_frame;
+}
+
+bool ViewerGraphicsWindow::addPrimitive(QString primitiveName) {
+    // Load  model
+    ModelLoader model;
+
+    QString  filepath = QString("../Data/Primitives/%1.obj").arg(primitiveName);
+    m_currentModel = model.LoadModel(filepath);
+
+    if (!m_currentModel.m_isValid) {
+        QString  filepath = QString("../Data/Primitives/%1.stl").arg(primitiveName);
```

Comment:
Should this just be filepath?

Reply:
I'm not sure what you're asking...?  It looks like we're trying multiple file types to find the right one.

Reply:
Because some models are not in obj format, it just for convenience, and I will unify the form in update.